test/integration_test/stork-specs/grafana/grafana.yaml

```diff
@@ -0,0 +1,69 @@
+apiVersion: v1
```

Comment:
This is nice! Do we plan to add this to the longevity and reliability test beds?